docs/build/build-transaction-construction.md

```diff
@@ -35,6 +35,29 @@ function from the Balances pallet will take:
 - `dest`: Destination address
 - `#[compact] value`: Number of tokens (compact encoding)
 
+**Serialized transaction format**
+
+Before being submitted, transactions are serialized. Serialized transactions are hex encoded SCALE-encoded bytes. The specific serialization is defined in the runtime and can change if the runtime is upgraded, but in general the serialization format can be described as follows:
+- Compact encoded number of SCALE-encoded bytes following this.
+- 1 bit: it is a 0 if no signature is present, or a 1 if it is.
+- 7 bits: the extrinsic version, it is equal to 4 in decimal.
+- If there is a signature:
+  - a SCALE encoded `sp_runtime::MultiAddress::Id<AccountId32, u32>` indicating the signer(s) of the transaction.
+  - a SCALE encoded `sp_runtime::MultiSignature::{SigningScheme}` with the signature\*.
+  - a SCALE encoded `sp_runtime::generic::Era` indicating for how long this transaction is valid.
+  - Compact encoded `u32` with the nonce.
+  - Compact encoded `u128` with the tip paid to the block producer.
+- The specific transaction parameters or call data, which consists of:
+  - 1 byte: the pallet index the transaction is calling into.
+  - 1 byte: the function in the pallet the transaction is calling.
+  - variable: the SCALE-encoded parameters required by the function being called.
```

Comment:
```suggestion
  - variable: the SCALE-encoded parameters required by the function being called.
- 4 bytes: Spec version of the runtime.
- 4 bytes: Transaction version of the runtime.
- 32 bytes: Genesis hash of the chain.
- 32 bytes: Block hash serving as the era reference. If the transaction is immortal, then this would be the genesis hash.
```